config/recipes/elastic-agent/README.asciidoc

```diff
@@ -31,6 +31,10 @@ Deploys two Elasticsearch clusters and two Kibana instances together with single
 
 Deploys Elastic Agent as a DaemonSet in Fleet mode with System and Kubernetes integrations enabled. System integration collects syslog logs, auth logs and system metrics (for CPU, I/O, filesystem, memory, network, process and others). Kubernetes integrations collects API server, Container, Event, Node, Pod, Volume and system metrics.
 
+===== System and Kubernetes integrations running as non-root - `fleet-kubernetes-integration-nonroot.yaml`
```

Comment:
This should also be listed in the documentation samples on the website

Reply:
@pebrc I think I've done what you're asking by adding it here

``` diff
diff --git a/docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc b/docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc
index 107464c4d..359f3e425 100644
--- a/docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc
+++ b/docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc
@@ -655,6 +655,14 @@ kubectl apply -f {agent_recipes}/fleet-kubernetes-integration.yaml
 ----
 Deploys Elastic Agent as a DaemonSet in Fleet mode with System and Kubernetes integrations enabled. System integration collects syslog logs, auth logs and system metrics (for CPU, I/O, filesystem, memory, network, process and others). Kubernetes integrations collects API server, Container, Event, Node, Pod, Volume and system metrics.

+=== System and Kubernetes integrations running as non-root
+
+[source,sh,subs="attributes"]
+----
+kubectl apply -f {agent_recipes}/fleet-kubernetes-integration-nonroot.yaml
+----
+The provided example is functionally identical to the previous section but runs the Elastic Agent processes (both the Elastic Agent running as the Fleet server and the Elastic Agent connected to Fleet) as a non-root user by utilizing a Daemonset to ensure directory and file permissions. *Note* The Daemonset itself must run as root to set up permissions.
+
 === Custom logs integration with autodiscover

 [source,sh,subs="attributes"]
```

Let me know if this is not what you were referring to.